.travis.yml

```diff
@@ -3,10 +3,11 @@ os:
   - linux
   - osx
 julia:
-  - 0.6
-  - 0.7
   - 1.0
+  - 1.1
 notifications:
   email: false
+before_script:
+  - julia -e 'using Pkg; Pkg.add(PackageSpec(name="MathOptInterface", rev="master"))'
```

Comment:
On CI we should use only released versions, shouldn't we?; If the changes in MOI are needed here, this has to wait until we can use the released version

Reply:
Yes, it will be removed before we merge